softwareComponents/configuration/rofibot.h

```diff
@@ -559,6 +613,18 @@ class Rofibot {
     }
 
     struct ModuleInfo {
+        ModuleInfo() = default;
+        ModuleInfo( const ModuleInfo& ) = default;
+        ModuleInfo( ModuleInfo&& o ) noexcept : module( std::move( o.module ) )
+        , inJointsIdx( std::move( o.inJointsIdx ) ), outJointsIdx( std::move( o.outJointsIdx ) )
+        , spaceJoints( std::move( o.spaceJoints ) ), position( std::move( o.position ) ) {}
+        ModuleInfo( atoms::ValuePtr< Module >&& m, const std::vector< int >& i, const std::vector< int >& o
+                    , const std::vector< int >& s, const std::optional< Matrix >& pos )
+        : module( std::move( m ) ), inJointsIdx( i ), outJointsIdx( o ), spaceJoints( s ), position( pos ) {}
+
```

Comment:
Zde jsem zmaten z formátování a velmi špatně se čte.

Reply:
Nechceme zavést nějaký defaultní clang-format? Sám používám svoji konfiguraci, ale pokud to sjednotíme, tak se klidně přizpůsobím.